src/TConsole.cpp

```diff
@@ -772,14 +777,31 @@ void TConsole::closeEvent( QCloseEvent *event )
 
     if( profile_name != "default_host" && ! mUserAgreedToCloseConsole )
     {
-        ASK: int choice = QMessageBox::question( this, "Exiting Session: Question", "Do you want to save the profile "+profile_name, QMessageBox::Yes|QMessageBox::No|QMessageBox::Cancel );
+        QCheckBox *cb = new QCheckBox( tr("Save the Window Layout before closing this profile.") );
```

Comment:
Ah, this is a bit more complex as "saving the windows layout" seems to be an application-wide _all or nothing affair_ and I am not sure how this works if multi-playing is being done.   Do it have to be asked before ANY profiles are saved and does the reloading only work _if the same profiles are subsequently reloaded_ and in that case do they all have to be loading in before we try to reload the layout???